Console: address creation fails with plan from brokered AddressSpace

After running "deploy-kubernetes.sh" from the EnMasse 0.17.0 release (no changes or parameters) and going into the EnMasse console,  I've clicked on the "Create address" button and have chosen type "queue".
In the next screen with the "plan" selection, I was surprised to see also an option from the "brokered-plans.yaml", namely "Brokered Queue" (I though that would be only available for the brokered address space).
After having selected that plan and created the address, the address stays in state "isReady":false,"phase":"Pending".

In the address controller log, there is this exception:
````
2018-02-27 17:51:56 WARN  DefaultChannelPipeline:151 - An exceptionCaught() event was fired, and it reached at the tail of the pipeline. It usually means the last handler in the pipeline did not handle the exception.
io.netty.handler.codec.DecoderException: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 474554202f20485454502f312e310d0a486f73743a203139322e3136382e39392e3130303a33303933390d0a436f6e6e656374696f6e3a206b6565702d616c6976650d0a557067726164652d496e7365637572652d52657175657374733a20310d0a557365722d4167656e743a204d6f7a696c6c612f352e3020285831313b204c696e7578207838365f363429204170706c655765624b69742f3533372e333620284b48544d4c2c206c696b65204765636b6f29205562756e7475204368726f6d69756d2f36342e302e333238322e313637204368726f6d652f36342e302e333238322e313637205361666172692f3533372e33360d0a4163636570743a20746578742f68746d6c2c6170706c69636174696f6e2f7868746d6c2b786d6c2c6170706c69636174696f6e2f786d6c3b713d302e392c696d6167652f776562702c696d6167652f61706e672c2a2f2a3b713d302e380d0a4163636570742d456e636f64696e673a20677a69702c206465666c6174650d0a4163636570742d4c616e67756167653a2064652d44452c64653b713d302e392c656e2d55533b713d302e382c656e3b713d302e370d0a0d0a
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:459)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1359)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:935)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:141)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:645)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:886)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: io.netty.handler.ssl.NotSslRecordException: not an SSL/TLS record: 474554202f20485454502f312e310d0a486f73743a203139322e3136382e39392e3130303a33303933390d0a436f6e6e656374696f6e3a206b6565702d616c6976650d0a557067726164652d496e7365637572652d52657175657374733a20310d0a557365722d4167656e743a204d6f7a696c6c612f352e3020285831313b204c696e7578207838365f363429204170706c655765624b69742f3533372e333620284b48544d4c2c206c696b65204765636b6f29205562756e7475204368726f6d69756d2f36342e302e333238322e313637204368726f6d652f36342e302e333238322e313637205361666172692f3533372e33360d0a4163636570743a20746578742f68746d6c2c6170706c69636174696f6e2f7868746d6c2b786d6c2c6170706c69636174696f6e2f786d6c3b713d302e392c696d6167652f776562702c696d6167652f61706e672c2a2f2a3b713d302e380d0a4163636570742d456e636f64696e673a20677a69702c206465666c6174650d0a4163636570742d4c616e67756167653a2064652d44452c64653b713d302e392c656e2d55533b713d302e382c656e3b713d302e370d0a0d0a
	at io.netty.handler.ssl.SslHandler.decodeJdkCompatible(SslHandler.java:1138)
	at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1194)
	at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489)
	at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428)
	... 16 more
````

--- a/templates/install/deploy-kubernetes.sh
+++ b/templates/install/deploy-kubernetes.sh
@@ -111,8 +111,6 @@ if [ $? -gt 0 ]; then
 fi
 
 runcmd "kubectl create sa enmasse-admin -n $NAMESPACE" "Create service account for address controller"
-runcmd "kubectl apply -f $ADDONS/standard-plans.yaml -n $NAMESPACE" "Create standard address space plans"
-runcmd "kubectl apply -f $ADDONS/brokered-plans.yaml -n $NAMESPACE" "Create brokered address space plans"
 
 create_self_signed_cert "kubectl" "address-controller.${NAMESPACE}.svc.cluster.local" "address-controller-cert"
 
@@ -136,8 +134,12 @@ then
 fi
 
 if [ "$MODE" == "singletenant" ]; then
+    runcmd "kubectl apply -f $ADDONS/standard-plans.yaml -n $NAMESPACE" "Create standard address space plans"
     runcmd "kubectl create sa address-space-admin -n $NAMESPACE" "Create service account for default address space"
     create_address_space "kubectl" "default" $NAMESPACE
+else
+    runcmd "kubectl apply -f $ADDONS/standard-plans.yaml -n $NAMESPACE" "Create standard address space plans"
+    runcmd "kubectl apply -f $ADDONS/brokered-plans.yaml -n $NAMESPACE" "Create brokered address space plans"
 fi
 
 runcmd "kubectl apply -f $ENMASSE_TEMPLATE -n $NAMESPACE" "Deploy EnMasse to $NAMESPACE"

